### ISSUE TITLE
Console spam

### ISSUE THREAD

**Issue by Dalsok** [ezeiger92/QuestWorld2#22]
Using questworld 2.6.4 (the latest release on github page)
Minecraft 1.12 Paper spigot

Not really an ERROR but it really does flod the Console
```
[16:20:08] [Server thread/INFO]: [QuestWorld] {type=SEA_LANTERN}
[16:20:08] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:09] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:09] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:10] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:12] [Server thread/INFO]: [QuestWorld] {type=STONE, damage=1}
[16:20:12] [Server thread/INFO]: [QuestWorld] {type=STONE, damage=1}
[16:20:12] [Server thread/INFO]: [QuestWorld] {type=SEA_LANTERN}
[16:20:13] [Server thread/INFO]: [QuestWorld] {type=STONE, damage=1}
[16:20:14] [Server thread/INFO]: [QuestWorld] {type=STONE, damage=1}
[16:20:15] [Server thread/INFO]: [QuestWorld] {type=STONE, damage=1}
[16:20:15] [Server thread/INFO]: [QuestWorld] {type=STONE, damage=1}
[16:20:16] [Server thread/INFO]: [QuestWorld] {type=STONE, damage=1}
[16:20:16] [Server thread/INFO]: [QuestWorld] {type=SEA_LANTERN}
[16:20:17] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:17] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:18] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:19] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:19] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:19] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:20] [Server thread/INFO]: [QuestWorld] {type=SEA_LANTERN}
[16:20:20] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:20] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:20] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:21] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:21] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:21] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:22] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:22] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:22] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:22] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:22] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:23] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:23] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:23] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:23] [Server thread/INFO]: [QuestWorld] {type=SEA_LANTERN}
[16:20:24] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:24] [Server thread/INFO]: [QuestWorld] {type=TORCH}
[16:20:24] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:25] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:26] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:26] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:26] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:26] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:26] [Server thread/INFO]: [QuestWorld] {type=SEA_LANTERN}
[16:20:27] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:27] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:27] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:27] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:27] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:28] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:28] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:29] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:29] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:29] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:29] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:29] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:30] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:30] [Server thread/INFO]: [QuestWorld] {type=IRON_ORE}
[16:20:30] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:30] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:30] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:30] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:30] [Server thread/INFO]: [QuestWorld] {type=STONE}
[16:20:31] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:31] [Server thread/INFO]: [QuestWorld] {type=DIRT}
[16:20:31] [Server thread/INFO]: [QuestWorld] {type=STONE}
```

**Comment by ezeiger92** [comment]
Well shoot, must have left a debug print in :x
Unfortunately I'm busy right now, might have time later today. Sorry!

**Comment by ezeiger92** [comment]
Should be fixed in the new release (2.6.5), let me know if this is still an issue.
